tests/test_sdds.py

```diff
@@ -110,6 +110,23 @@ def test_read_header(self):
         for definition in definitions:
             assert definition.type == test_data[definition.name]
 
+    def test_read_header_optionals(self):
+        test_head = b"""
+        SDDS1
+        !# little-endian
+        &description text="Momentum aperture search", contents="momentum aperture", &end
+        &parameter name=Step, type=long, &end
+        &parameter name=SVNVersion, description="SVN version number", type=string, fixed_value=28096M, &end
+        &column name=ElementName, type=string,  &end
+        &column name=s, units=m, type=double,  &end
+        &column name=ElementType, type=string,  &end
+        &column name=ElementOccurence, type=long,  &end
+        &column name=deltaPositiveFound, type=short,  &end
+        &column name=deltaPositive, symbol="$gd$R$bpos$n", type=double,  &end
+        &data mode=binary, &end
+        """
+        version, definitions, _, data = _read_header(io.BytesIO(test_head))
```

Comment:
I feel like we're testing that the reader doesn't crash but not that it reads the correct information here

Reply:
now tests content as well